resources/analysisTools/roddyTools/wrapInScript.sh

```diff
@@ -191,37 +191,53 @@ runEnvironmentSetupScript() {
 # Set the "RODDY_SCRATCH" variable and directory from the predefined "RODDY_SCRATCH" variable or the "defaultScratchDir" variable.
 # Die if the resulting directory is not accessible (executable).
 setupRoddyScratch() {
-  if [[ "${RODDY_SCRATCH:-}" == "" ]]; then
-    throw 200 "Undefined RODDY_SCRATCH variable."
-  elif [[ ! -d ${RODDY_SCRATCH} ]]; then
-    mkdir -p ${RODDY_SCRATCH}
-  fi
-  if [[ ! -x "$RODDY_SCRATCH" ]]; then
-    throw 200 "Cannot access RODDY_SCRATCH=$RODDY_SCRATCH"
-  fi
-  echo "RODDY_SCRATCH is set to ${RODDY_SCRATCH}"
+    if [[ "${RODDY_SCRATCH:-}" == "" ]]; then
+        throw 200 "Undefined RODDY_SCRATCH variable."
+    elif [[ ! -d ${RODDY_SCRATCH} ]]; then
+        mkdir -p ${RODDY_SCRATCH}
+    fi
+    if [[ ! -x "$RODDY_SCRATCH" ]]; then
+        throw 200 "Cannot access RODDY_SCRATCH=$RODDY_SCRATCH"
+    fi
+    echo "RODDY_SCRATCH is set to ${RODDY_SCRATCH}"
+}
+
+sourceBaseEnvironment() {
```

Comment:
Maybe put in a comment for what this is?

Reply:
I thought that not necessary, because the name says it. I was considering renaming to `sourceBaseEnvironmentScript`. Would that make it clearer, or just wordier?